mtags/src/main/scala-2/scala/meta/internal/pc/completions/ScalaCliCompletions.scala

```diff
@@ -11,10 +11,12 @@ trait ScalaCliCompletions {
     def unapply(path: List[Tree]): Option[String] =
       path match {
         case Nil =>
-          CoursierComplete.isScalaCliDep(pos.lineContent.replace(CURSOR, ""))
+          CoursierComplete.isScalaCliDep(
+            pos.lineContent.replace(CURSOR, "").take(pos.column - 1)
+          )
         case (_: PackageDef) :: Nil if pos.source.file.path.endsWith(".sc") =>
           CoursierComplete.isScalaCliDep(
-            pos.lineContent.replace(CURSOR, "").take(pos.column)
+            pos.lineContent.replace(CURSOR, "").take(pos.column - 1)
```

Comment:
Could you add the test case for it? of by one errors are a bit tricky.

Reply:
Actually it's not an error, but I haven't realised before that `pos.column` in Scala 2 starts counting from 1 😮 not from 0 like in Scala 3. So in `ab@@cd` `pos.column == 3`, hence the `- 1`. Should I add a comment in the code?